resources/views/pages/import.blade.php

```diff
@@ -2,47 +2,34 @@
 
 @section('main_content')
 
-@if ($type === \Chompy\ImportType::$rockTheVote)
-    <div>
-        <h1>Rock The Vote</h1>
-        <p>Creates/updates users and their voter registration post via CSV from Rock The Vote.</p>
-        <h4>Users</h4>
-        <dl class="dl-horizontal">
-            <dt>Email subscriptions</dt><dd>{{ $userConfig['email_subscription_topics'] }}</dd>
-            <dt>Reset email enabled</dt><dd>{{ $resetConfig['enabled'] ? 'true' : 'false'}}</dd>
-            <dt>Reset email type</dt><dd>{{ $resetConfig['type'] }}</dd>
-        </dl>
-        <h4>Posts</h4>
-        <dl class="dl-horizontal">
-            <dt>Action ID</dt><dd>{{ $postConfig['action_id'] }}</dd>
-            <dt>Type</dt><dd>{{ $postConfig['type'] }}</dd>
-            <dt>Source</dt><dd>{{ $postConfig['source'] }}</dd>
-        </dl>
-        <form action={{url('/import')}} method="post" enctype="multipart/form-data">
-            {{ csrf_field() }}
-            <div class="form-group">
-                <div class="input-group">
-                    <label class="input-group-btn">
-                        <span class="btn btn-default">
-                            Select CSV <input type="file" name="upload-file" style="display: none;" multiple>
-                        </span>
-                    </label>
-                    <input type="hidden" name="import-type" value={{ app('request')->input('type') }}>
-                    <input type="text" class="form-control" readonly>
-                </div>
+<div>
+    @if ($type === \Chompy\ImportType::$rockTheVote)
+    @include('pages.partials.rock-the-vote', ['config' => $config])
+    @endif
+    <form action={{ app('request')->url() }} method="post" enctype="multipart/form-data">
```

Comment:
This should probably be explicitly pointed at the "store" route with `route('import.store')`